app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -152,12 +152,13 @@ def process_valid_webauthn(form)
         platform_authenticator: form.platform_authenticator?,
         enabled_mfa_methods_count: mfa_user.enabled_mfa_methods_count,
       )
-      Funnel::Registration::AddMfa.call(current_user.id, 'webauthn', analytics)
       mark_user_as_fully_authenticated
       handle_remember_device
       if form.platform_authenticator?
+        Funnel::Registration::AddMfa.call(current_user.id, 'webauthn_platform', analytics)
         flash[:success] = t('notices.webauthn_platform_configured')
       else
+        Funnel::Registration::AddMfa.call(current_user.id, 'webauthn', analytics)
         flash[:success] = t('notices.webauthn_configured')
       end
       user_session[:auth_method] = 'webauthn'
```

Comment:
This line should change as well, but https://github.com/18F/identity-idp/pull/6824/files#diff-aef799e3b0cf957f5f75004e1b33aa1172334e0bd635ed98f68ed6b235fc3712R2 needs to go live first to avoid breaking during the deploy.